Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/shopify/use-product.md

# useProduct composable

> `useProduct` composition API function is responsible, as its name suggests, for interactions with products from your eCommerce. This function returns the following values:

Reviewer: ```suggestion
`useProduct` composition API function allows for loading the products. This function returns the following values:
```
